test/lib/source-handlers/bitbucket-server/bitbucket-server.test.ts

```diff
@@ -0,0 +1,16 @@
+import { listBitbucketServerProjects } from '../../../../src/lib/source-handlers/bitbucket-server/list-projects';
+
+jest.setTimeout(60000);
+
+describe('listBitbucketServerProjects script', () => {
+  it('list projects', async () => {
+    process.env.BITBUCKET_SERVER_TOKEN = process.env.BBS_TOKEN;
```

Comment:
are these variables set in Circle already & saved to the password vault for this repo too?

Reply:
Yes for CircleCI, just added to the vault as well.